.vscode/settings.json

```
@@ -35,5 +35,5 @@
     "zxcvbn"
   ],
   "rust-analyzer.cargo.targetDir": true,
-  "rust-analyzer.cargo.features": ["all"]
+  "rust-analyzer.cargo.features": ["wasm", "internal"]
```

Ah, you have to set the `all` outside the array, just as a plain string for it to work:
```suggestion
  "rust-analyzer.cargo.features": "all"
```

---

Ty!